operator/config/manifests/bases/forklift-operator.clusterserviceversion.yaml

```diff
@@ -101,18 +100,12 @@ spec:
     * Controller, to coordinate migration processes.
     * UI, the web console to manage migrations.
     * Validation, a service to validate migration workflows.
-    * Must-gather-api, a service to generate targeted must-gather archives.
 
     ### Compatibility
-    Forklift 2.2 is supported on OpenShift 4.9
-
-    Forklift 2.3 is supported on OpenShift 4.10 and 4.11
-
-    Forklift 2.4 is supported on OpenShift 4.11 and 4.12
     
     Forklift 2.5 is supported on OpenShift 4.12 to 4.15
 
-    Forklift 2.6 is supported on OpenShift 4.14 and 4.15
+    Forklift 2.6 is supported on OpenShift 4.14 to 4.16
```

Comment:
it's a bit weird to mention OpenShift 4.16 before it's released (and before we test Forklift on it), let's hold it